### ISSUE TITLE
Error: This document doezs not have a Zenodo reference

### ISSUE THREAD

**Issue by brokentool** [gsautter/goldengate-qualitycontrol#71]
in: `36805-40310-1-PB.pdf.imf`

Even after re-check. Not sure what to do in this case

**Comment by gsautter** [comment]
The prerequisite for an article to go into Zenodo is complete metadata ... so if there are any errors in that regard, you need to store the document and close it so the Zenodo uploader can take over ... if you open the document again later, the Zenodo deposition number should be there.

**Comment by angel-s-slavchev** [comment]
Document:
ejt-606_capmos-filhos-fernandes-cordoso.pdf.imf

I tried to complete the metadata as best as possible - used extract from file and filled in all I dould find. It validates fine.

What other field is required for a successful Zenodo submission?
Which filelds are required for that in general?

**Comment by gsautter** [comment]
EJT 606 is kind of an accident ... both the PDF and the PDF/A versions of that one got converted and uploaded to the server, and since the two PDFs are different at the byte level, hash based duplicate detection doesn't kick in. Now Zenodo de-duplicates by DOI, which is the same for both PDFs, so it refuses accepting the second PDF once the first one is in ... preciously little we can do about this, and nothing we should do, either, because we don't want to duplicate the Zenodo content.

Bottom line: please ignore this issue for now, we'll take care of removing the duplicate.

**Comment by angel-s-slavchev** [comment]
Good enough for us. The important thing is that the process of fixing this error generally works.

**Comment by brokentool** [comment]
![scrn28](https://user-images.githubusercontent.com/22291049/75862468-ae48a580-5e07-11ea-83fd-f7565e788eef.png)

the highlighted selection is with documents that only have a lack of zenodo reference left, after QC. the ones I checked appear to have metadata with no errors

**Comment by gsautter** [comment]
For the articles that show up in the collection statistics (5 from this list), I'm getting a "DOI already in Zenodo" error for each one, even though some DOIs don't seem to find anything when searching for them in Zenodo via the UI ... definitely _not_ a QC issue, meaning to say @brokentool doesn't need to take care of these.

**Comment by gsautter** [comment]
The others have either been uploaded by now, or do not get through to the XML stage of our system at all. The former would remove the error via a "Re-Check All", the latter would most likely hint to some other metadata issue.

Can you copy the list to some Excel for me? Typing the file names off the screenshot is a bit too error prone ...

**Comment by brokentool** [comment]
sure thing, however it may need to wait until tomorrow, as I am tackling a 330 page monograph in PhytoKeys, that needs to be published today. Will try to do it by the end of today, but can't promise

**Comment by gsautter** [comment]
Sure thing, take your time with that monograph ... I'm not exactly dying from boredom anyway.
